packages/cli/core/src/commands/ui/deploy.ts

```diff
@@ -107,6 +107,36 @@ export default class Deploy extends CLICommand {
         'Create a new Hosted Page according to the configuration in the file located at "./configs/myconfig.json"',
     },
   ];
+  public static summary = `The target "config" must contain the following parameters:
+  {
+    "name": "The name of the hosted search page.",
+    "dir": "The directory of the hosted search page.",
+    "htmlEntryFile": {
+      "path": "The path to an HTML file containing the HTML markup of the hosted page."
+    },
+    "javascriptEntryFiles": [
+      {
+        "path": "The path to a bundled Javascript file.",
+        "isModule": "Whether the inline code should be treated as a JavaScript module. If this property is true, the type property will be set to "module" on the script tag."
+      }
+    ],
+    "javascriptUrls": [
+      {
+        "path": "The URL of the JavaScript source file.",
+        "isModule": "Whether the inline code should be treated as a JavaScript module. If this property is true, the type property will be set to "module" on the script tag."
+      }
+    ],
+    "cssEntryFiles": [
+      {
+        "path": "The path to a bundled CSS file."
+      }
+    ],
+    "cssUrls": [
+      {
+        "path": "The URL of the CSS stylesheet."
+      }
+    ]
+  }`;
```

Comment:
```ts
export class MyCommand extends Command {
  static summary = 'A brief overview of your command.'
  static description = `
An in-depth description of the command.
It can be multiline.
`
```
https://oclif.io/docs/commands

You will probably need both, and the long stuff should be in the `description`.
I reckon that `summary` will be used to describe the command when checking a topic etc and `description` for the more verbose stuff.

Reply:
read too fast